etc/tor.profile

```diff
@@ -0,0 +1,47 @@
+# Firejail profile for tor
+# This file is overwritten after every install/update
+# Persistent local customizations
+include /etc/firejail/tor.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
```

Comment:
Could `blacklist /tmp/.X11-unix` be a useful addition? Another question is how useful `netfilter` is in this context.